packages/framework/src/Form/Admin/Customer/User/CustomerUserFormType.php

```diff
@@ -112,6 +115,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'disabled' => !$options['allowEditSystemData'],
             ]);
 
+        $builderSystemDataGroup
+            ->add('salesRepresentative', ChoiceType::class, [
+                'required' => false,
+                'choices' => $this->salesRepresentativeFacade->getAll(),
+                'choice_label' => 'fullName',
+                'choice_value' => 'id',
+                'label' => t('Sales representative'),
+                'attr' => [
+                    'class' => 'js-toggle-opt-group',
+                    'data-js-toggle-opt-group-control' => '.js-toggle-opt-group-control',
+                ],
```

Comment:
this is not necessary here, or am I missing something?

Reply:
you are right